### DISCUSSION TITLE
Who can help me using the kleinanzeigen-bot

### DISCUSSION THREAD

**Discussion by Munaivert** [Second-Hand-Friends/kleinanzeigen-bot#165] [Q&A] [answered]
Dear forum,

I am reaching out for some help with the Kleinanzeigen Bot. I am not very experienced with using Python, and I would really appreciate a step-by-step explanation on how to set up and use the bot. I've been trying to follow some online tutorials, but I'm still having trouble.

For example, when I tried to run the bot using the command "kleinanzeigen-bot verify" in the Windows command prompt, I received a syntax error. I asked for help and was told to use the command "python kleinanzeigen-bot.py verify" instead. However, when I tried this, I got an error saying that Python was not found. I was then advised to add Python to my system path, but I'm not sure how to do this ( I downloaded it several times and it seems to work perfect)

If anyone could provide some guidance on how to set up and use the Kleinanzeigen Bot, including how to add Python to my system path, I would be extremely grateful.

Thank you in advance for your help.

Sincerely,
Munaivert

**Comment by sebthom** [comment, marked as answer]
You should be able to follow this instruction: https://github.com/Second-Hand-Friends/kleinanzeigen-bot#installation

You should not need to separately install python. If you get a syntax error this might be a bug, then please post the version of the bot and the complete error message here.

**Reply by Fedfred** [reply]
HI, I tried to install it via Windows, but it doesn't work.

The following error message appears in the terminal:

> curl -L https://github.com/Second-Hand-Friends/kleinanzeigen-bot/releases/download/latest/kleinanzeigen-bot-windows-amd64.exe -o kleinanzeigen-bot.exe
Invoke-WebRequest : Es wurde kein Parameter gefunden, der dem Parameternamen "L" entspricht.
In Zeile:1 Zeichen:6
+ curl -L https://github.com/Second-Hand-Friends/kleinanzeigen-bot/rele ...
+      ~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-WebRequest], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.InvokeWebRequestCommand

**Reply by sebthom** [reply]
This happens when you are trying to executed the curl command from PowerShell, not the Windows command prompt (Windows Eingabeaufforderung). Maybe that helps:

- https://www.giga.de/downloads/windows-10/tipps/windows-10-eingabeaufforderung-oeffnen/
- https://www.delftstack.com/de/howto/powershell/run-curl-command-via-powershell/

**Reply by Fedfred** [reply]
But if i use the normal CMD Terminal, then i get this error code:

curl -L https://github.com/Second-Hand-Friends/kleinanzeigen-bot/releases/download/latest/kleinanzeigen-bot-windows-amd64.exe -o kleinanzeigen-bot.exe
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - Die Sperrfunktion konnte keine Sperrprüfung für das Zertifikat durchführen.


I have tested it on several Notebooks.

**Reply by sebthom** [reply]
You can also just download the exe with your browser from here https://github.com/Second-Hand-Friends/kleinanzeigen-bot/releases/tag/latest

**Reply by Fedfred** [reply]
Okay, thank you. Now it starts but after a few seconds it close the window of Kleinanzeigen Bot
